attempt_6_animals/simple-reference-animals.squiggle

```diff
@@ -126,16 +111,260 @@ improvement_as_proportion_of_lifetime = 0.05 to 0.5
 value_fwi_fish = 
   fish_potentially_helped * 
 	improvement_as_proportion_of_lifetime * 
-	(value_happy_salmon_year_in_human_qalys / salmon_estimates.lifetime)
+	(salmon_estimates.value_year_in_human_qalys / salmon_estimates.lifetime)
 
 value_of_shrimp_in_fish = (0.3 to 1)
 // ^ very uncertain, subjective
 
 value_fwi_shrimp = 
   shrimp_potentially_helped * 
 	improvement_as_proportion_of_lifetime * 
-	(value_happy_salmon_year_in_human_qalys / salmon_estimates.lifetime) *
+	(salmon_estimates.value_year_in_human_qalys / salmon_estimates.lifetime) *
   value_of_shrimp_in_fish
 
 value_fwi_so_far = value_fwi_fish + value_fwi_shrimp
-value_fwi_so_far
+proportion_fwi_in_2022 = 1/4 to 1/2
+value_fwi  = value_fwi_so_far * proportion_fwi_in_2022
+fwi_item = {
+  name: "Fish Welfare Inititative (2022)",
+	value: value_fwi -> ss 
+}
+
+/*
+## Open Wing Alliance
+
+Sources: 
+- <https://animalcharityevaluators.org/charity-review/the-humane-league/#comprehensive-review>
+- <https://thehumaneleague.org/article/owa-year-in-review-2022>
+
+What am I estimating: Impact of corporate commitments that OWA made happen in 2022.
+
+Notes on the estimation: The number of corporate campaigns is from OWA's own reporting. The number of chickens affected and the improvement per chicken is particularly speculative here.
+
+Key simplification: I am using the welfare numbers for meat chickens, rather than for egg-laying chicken. This will give inaccurate results, but hopefully not by many orders of magnitude.
+*/
+
+num_corporate_commitments = 150 to 200 
+// ^ "nearly 200 commitments", from https://thehumaneleague.org/article/owa-year-in-review-2022
+average_num_chickens_per_year_per_commitment = 5k to 20k 
+// this: <https://thehumaneleague.org/article/victory-toridoll-releases-a-global-cage-free-commitment?ms=c_blog>
+// mentions 40k, but also seems to be the most impressive one for 2022 (?)
+frequency_promises_are_fulfilled = 0.4 to 0.9
+average_duration_fulfilled_commitments = 1 to 10 // years. Could be more?
+average_reduction_in_suffering_of_commitment_per_chicken = 0.05 to 0.3 // pulled out of my ass.
+
+value_owa_2022 = num_corporate_commitments * 
```

Comment:
Here you're using one inside-view model, to represent the entire org. I'd also recommend mixing this with maybe 5% mass, on a very wide intuitive estimate, or similar. Otherwise you could really be hurt when scored. (You need to watch the tails)

Reply:
Good point, done.